docs/getting-started.md

```diff
@@ -287,6 +287,16 @@ This will build and place the two Firecracker binaries at:
 - `build/cargo_target/${toolchain}/debug/firecracker` and
 - `build/cargo_target/${toolchain}/debug/jailer`.
 
+If you would like to test a new feature and work with
+dependencies on libraries located in private git repos, you
+can use the `--pub-key` and `--priv_key` flags to specify the
+path to your public and private ssh keys. Both of them are
+required for git authentication when fetching the repositories.
+
+```bash
+tools/devtool build --pub-key ~/.ssh/id_rsa.pub --priv-key ~/.ssh/id_rsa
```

Comment:
Since you always need both, maybe have a single param that takes some sort of _pair_ value or at least a validation to explicitly fail or output warning when only one is provided.

Reply:
Good point. I think that adding a flag that takes a pair value would work best in this case.